### PR TITLE
Mainnet environment & validator keys

### PR DESCRIPTION
**Pull request by tkporter**
Adds `mainnet` environment with the new validator keys

TODO:
* Figure out the exact blocks to finality for all mainnets
* Actually set up mainnet infrastructure
* Add RPC urls to GCP secret manager

### CODE REVIEW

File: typescript/infra/config/environments/mainnet/validators.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    ],
+  },
+  ethereum: {
+    threshold: 2,
+    validators: [
+      {
+        address: '0x7c1d96c9e72c41b84d74095dc2e72b979e933904',
+        checkpointSyncer: {
+          type: CheckpointSyncerType.S3,
+          bucket: s3BucketName('kovan', 0),
```
Comment (nambrot) [inline typescript/infra/config/environments/mainnet/validators.ts:52]: Is this kovan here intended?

File: typescript/sdk/src/chain-metadata.ts [hunk 2]
```diff
@@ -17,6 +17,21 @@ export const ethereum: ChainMetadata = {
...
+};
+
+export const optimism: ChainMetadata = {
+  id: 0x6f70, // b'op' interpreted as an int
+  finalityBlocks: 1,
+};
+
+export const bsc: ChainMetadata = {
+  id: 0x627363, // b'bsc' interpreted as an int
+  finalityBlocks: 7,
```
Comment (nambrot) [inline typescript/sdk/src/chain-metadata.ts:32]: Isn't BSC PoA basically?

File: typescript/sdk/src/chain-metadata.ts [hunk 3]
```diff
@@ -17,6 +17,21 @@ export const ethereum: ChainMetadata = {
   finalityBlocks: 7,
 };
 
+export const arbitrum: ChainMetadata = {
+  id: 0x617262, // b'arb' interpreted as an int
+  finalityBlocks: 1,
```
Comment (nambrot) [inline typescript/sdk/src/chain-metadata.ts:22]: We probably don't have a good setup here, but is that how rollup finality works?

Comment (tkporter) [reply typescript/sdk/src/chain-metadata.ts:22]: This ones tougher - will have to do some reading, but any insight here would be helpful. I'm no expert but I assume we are trusting the RPC endpoint to be accurate and not falling for any fraud, in which case I'm pretty sure we don't need to worry about reorgs. Definitely fuzzy here though and will read some more before doing the agent deploy

Comment (nambrot) [reply typescript/sdk/src/chain-metadata.ts:22]: https://developer.offchainlabs.com/docs/finality is the relevant page. I think long story short, we should just assume instant finality, everything else is going to be too complicated

**Comment by asaj** [comment]
Does this fix https://github.com/abacus-network/abacus-monorepo/issues/581?

File: typescript/sdk/src/chain-metadata.ts (hunk 2 above)
Comment (asaj) [reply typescript/sdk/src/chain-metadata.ts:32]: @tkporter can you add a ticket for the above?

**Pull request merged**